src/Connector.js

```diff
@@ -190,7 +190,7 @@ async function removeDeviceFromIoTAgent(iotAgentUrl, id) {
 
   const promises = sensors.devices.map(async (sensor) => {
     url = `${iotAgentUrl}/iot/devices/${sensor.device_id}`;
-    await request.delete({ url, headers, json: true });
+    return request.delete({ url, headers, json: true });
```

Comment:
About commit message 28e3882fc3dfcb4242ba9ba3a017ae3dc3749199:
remove blank space
...thus **'map'**

Reply:
The commit title is too specific. Remember that the title should be saying if you fixed a bug, a refactor, a new feature and not what you've changed on code (if the developer wants to see what has changed on code it can see the diff). Do you agree?

Reply:
Totally agree, will be changing the commit message! :+1: